packages/tezos-stdlib-unix/tezos-stdlib-unix.16.0~rc1/opam

```diff
@@ -0,0 +1,38 @@
+opam-version: "2.0"
+maintainer: "contact@tezos.com"
+authors: ["Tezos devteam"]
+homepage: "https://www.tezos.com/"
+bug-reports: "https://gitlab.com/tezos/tezos/issues"
+dev-repo: "git+https://gitlab.com/tezos/tezos.git"
+license: "MIT"
+depends: [
+  "dune" { >= "3.0" }
+  "base-unix"
+  "tezos-error-monad" { = version }
+  "tezos-lwt-result-stdlib" { = version }
+  "tezos-event-logging" { = version }
+  "tezos-stdlib" { = version }
+  "data-encoding" { >= "0.7.1" & < "1.0.0" }
+  "lwt" { >= "5.6.0" }
+  "ipaddr" { >= "5.0.0" & < "6.0.0" }
+  "re" { >= "1.9.0" }
+  "ezjsonm" { >= "1.1.0" }
+  "ptime" { >= "1.0.0" }
+  "mtime" { >= "1.4.0" }
```

Comment:
```suggestion
  "mtime" { >= "1.4.0" & < "2.0.0" }
```

The conversion functions were removed in 2.0.0: https://github.com/dbuenzli/mtime/blob/master/CHANGES.md

Reply:
Applied in 06703111c68508673b5208e659144a57ad1eb046